src/components/Sigmet/SigmetEditMode.jsx

```diff
@@ -75,21 +74,42 @@ class SigmetEditMode extends PureComponent {
     }
   }
 
-  verifySigmetDebounced (sigmetObject) {
+  verifySigmet (sigmetObject) {
     const { dispatch, actions } = this.props;
     dispatch(actions.verifySigmetAction(sigmetObject));
   }
 
-  componentWillReceiveProps (nextProps) {
-    if (nextProps.sigmet && this.props.sigmet && nextProps.geojson && this.props.geojson) {
-      if (nextProps.sigmet !== this.props.sigmet || nextProps.geojson !== this.props.geojson) {
-        this.verifySigmetDebounced(nextProps.sigmet);
+  componentDidUpdate (prevProps) {
+    if (prevProps.sigmet && this.props.sigmet && prevProps.geojson && this.props.geojson) {
+      /* The issue with the looping updates of the TAC message in the sigmet, is because the tac property is inside the sigmet object.
+        When you update the tac, the sigmet is different triggering a new update.
```

Comment:
- The problem is not the TAC being inside of the props. The problem is that we change the props in willReceiveProps or didUpdate. The previous introduction of the debounce workaround should have made us think why we needed it. The data flow should come in via the props or we should use a copy in the state of SigmetEditMode.